### DISCUSSION TITLE
KVM host in Alert State with Alert message Can not find network

### DISCUSSION THREAD

**Discussion by amitmvyas** [apache/cloudstack#9712] [Q&A]
##### ISSUE TYPE

 * Other

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.1
~~~

##### CONFIGURATION

Advance Network with following configuration.
1. 2 x 100G in Bond name "server_bond0"
2. 2 x 100G in Bond name "storage_bond0"
3. 1 x 1G Management NIC
4. server_bond0 is a trunk port with passing multiple VLANs
5. storage_bond0 is in access VLAN
6. below is the output of "**brctl show**"
    bridge name              bridge id                      STP enabled     interfaces
    **cloud0**                       8000.3271cbfe51f6        no
     **internet_br0**            8000.723f8add10ca       no                   internet4090
     **management_br0**   8000.624ab6643d88     no                   enp33s0f0
     **storage_br0**             8000.8a237b1368be      no                   storage18

##### OS / ENVIRONMENT

Ubuntu 22.04.4 LTS and 24.04.4 LTS

##### SUMMARY

While creating the Zone at Step No.4 -> Physical Network, I am providing following information.
1. Traffic types 
2. Guest -> keeping it blank
3. Management -> management_br0
4. Public -> internet_br0
5. Storage -> internet_br0
![image](https://github.com/user-attachments/assets/52d678cb-40a3-4667-9724-1965d84b06f2)

##### STEPS TO REPRODUCE

~~~
I feel using the same kind of config you may be able to reproduce the problem.
~~~

##### EXPECTED RESULTS

~~~
Host should get added without any error message.
~~~

##### ACTUAL RESULTS

~~~
Getting below error message and host goes in to **Alert** state.
Incorrect Network setup on agent, Reinitialize agent after network names are setup, details : Can not find network: internet_br0
~~~

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by alexandremattioli** [comment]
@amitmvyas can you paste your netplan config here please?

**Reply by amitmvyas** [reply]
Below is the netplan config details

network:
  version: 2
  renderer: networkd
  ethernets:
    enp33s0f0:
      dhcp4: false
    enp33s0f1:
      dhcp4: false
    enp193s0f0np0:
      dhcp4: false
      mtu: 9124  
    enp193s0f1np1:
      dhcp4: false
    enp65s0f0np0:
      dhcp4: false
      mtu: 9124
    enp65s0f1np1:
      dhcp4: false
      mtu: 9124
  bonds:
    server_bond0:
      interfaces: [enp193s0f0np0, enp65s0f0np0]
      mtu: 9124
      parameters:
        mode: 802.3ad
        transmit-hash-policy: layer3+4
        lacp-rate: fast
        mii-monitor-interval: 100
    storage_bond0:
      interfaces: [enp193s0f1np1, enp65s0f1np1]
      mtu: 9124
      parameters:
        mode: 802.3ad
        transmit-hash-policy: layer3+4
        lacp-rate: fast
        mii-monitor-interval: 100
  vlans:
    internet4090:
      id: 4090
      link: server_bond0
    storage18:
      id: 18
      link: storage_bond0
  bridges:
    management_br0:
      interfaces: [enp33s0f0]
      dhcp4: true
    cloud_br0:
      interfaces: [internet4090]
      addresses: [xxx.xxx.xxx.xxx/23]
    storage_br0:
      interfaces: [storage18]
      addresses: [xxx.xxx.xxx.xxx/24]

**Reply by amitmvyas** [reply]
Please read cloud_br0 as internet_br0.

**Comment by alexandremattioli** [comment]
>Guest -> keeping it blank_
This can't be kept blank, otherwise ACS won't know in which bridge to place your guest network VLANs
>Management -> management_br0
>Public -> internet_br0
Those should be fine
>Storage -> internet_br0
Did you mean "storage_br0" ?

**Reply by amitmvyas** [reply]
Hi, 
Yes, its storage_br0 and not internet_br0.

I would like to know, do I have create new brdige for guest or I can use any existing one ?

**Comment by amitmvyas** [comment]
While configuring the zone I am following below steps.
1. Zone Type - Core
2. Core Zone Type - Advance (Security groups is - Off)
3. Zone Details - any name, IPv4 DNS: 8.8.8.8, Internal DNS, 192.168.1.53, Hypervisor: KVM
4. Network - 
    a. Network Name - Physical Network 1
    b. Isolation method - VLAN
    c. Traffic Types - Guest (guest_br0), Management (management_br0), Public (internet_br0), Storage (storage_br0)

Rest of the steps are standard, once I click on launh zone, I am getting the message  "Incorrect Network setup on agent, Reinitialize agent after network names are setup, details : Can not find network: guest_br0"

**Comment by DaanHoogland** [comment]
closed this for lack of interaction, please reopen or open a new item if applicable
